contrib/platform/src/com/sun/jna/platform/win32/Advapi32Util.java

```diff
@@ -593,9 +609,28 @@ public static boolean registryKeyExists(HKEY root, String key) {
 	 * @return True if the value exists.
 	 */
 	public static boolean registryValueExists(HKEY root, String key,
-			String value) {
+											  String value) {
+		return registryValueExists(root, key, value, WinNT.REG_NONE);
```

Comment:
Please don't use `WinNT.REG_NONE` for this use-case. `WinNT.REG_NONE` is a type, not an access level. There is no "NONE" access level for the registry. I would just use 0 here. I would not invent a constant.